### ISSUE TITLE
scrolling buildings problem

### ISSUE THREAD

**Issue by zdechlak** [forkserf/forkserf#350]
by clicking on the scroll icon - if there is a building under the window, it displays it - instead of scrolling

https://user-images.githubusercontent.com/2237225/221092350-3a00758f-99ea-4006-a4c7-8055cd68946a.mp4

**Comment by tlongstretch** [comment]
yes, clicking "through" popups is a known issue I need to resolve.  I don't think this is new it has been happening forever for me

**Comment by tlongstretch** [comment]
What do you mean by "clicking the scroll icon" ?  What scroll icon?

**Comment by zdechlak** [comment]
> What do you mean by "clicking the scroll icon" ? What scroll icon?

![image](https://user-images.githubusercontent.com/2237225/221337308-8a57c803-c76d-450a-a3c9-d1510a209fde.png)

**Comment by zdechlak** [comment]
> yes, clicking "through" popups is a known issue I need to resolve. I don't think this is new it has been happening forever for me

It worked well for me earlier, it started not working in version 0.6.2.a
